vscode/vscode.md

```diff
@@ -0,0 +1,103 @@
+This section outlines how to set up the application in the VS CODE IDE.
+
+## Prerequisites
+
+* Java SE 8, Java SE 11 or Java SE 17 is required.
+* Payara Server 5 is required. You can download Payara Server 5 from [here](https://www.payara.fish/downloads/).
+* The VS CODE IDE is required. You can download the VS CODE from [here](https://code.visualstudio.com/download).
+
+## Download
+
+[Download](https://github.com/eclipse-ee4j/cargotracker/archive/master.zip) the source code zip file and expand it somewhere in your file system. Note that this is a Maven project.
+
+## VSCODE IDE Setup
+
+Before going through this tutorial, you must install the following extensions from the Visual Studio Code marketplace:
+
+* [Java Extension Pack, by Microsoft](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack)
+* [Payara Tools, by Payara](https://marketplace.visualstudio.com/items?itemName=Payara.payara-vscode)
+
+You can install extensions from Extensions view (Ctrl + Shift + X) in the activity bar of VS Code:
+
+![Payara](image/payara_server.png)
+ 
+Let’s start:
+
+ * Open the command pallet using Ctrl + Shift + P, type Maven and select the Create Maven Project option:
+
+  ![Maven](image/maven.png)
+
+ * Or in the explorer sidebar expand Maven Projects, Select the + sign to create a Maven Project:
+
+  ![Maven](image/maven1.png)
+
+ * Select More… option to find archetype available in remote catalog:
+
+  ![More](image/more.png)
+
+ * Search for the javaee8-essentials-archetype or you may select any other archetype:

```

Comment:
There is no need to start from an Archetype. As long as the correct extensions are installed, simply opening the folder is sufficient.

Reply:
Should I delete this line?